### PR TITLE
feat(content): reuse of .22 casings

### PR DESCRIPTION
**Pull request by Lamandus**
### Required

- [X] I wrote the PR title in [conventional commit format](https://docs.cataclysmbn.org/en/contribute/changelog_guidelines/).
- [X] I ran the [code formatter](https://docs.cataclysmbn.org/en/contribute/contributing/#code-style).
- [X] I linked any relevant issues using [github keyword syntax](https://docs.cataclysmbn.org/en/contribute/contributing/#pull-request-notes) so it can be closed automatically.
- [X] I have [committed my changes to new branch that isn't `main`](https://docs.cataclysmbn.org/en/contribute/contributing/#make-your-changes) so it won't cause conflict when updating `main` branch later.

## Purpose of change
.22 was in an odd place. already sub par, it wasn't a very well used ammo to begin with, having it "not able to handload" it doesn't help the case. There are videos of people reloading .22 with oxy or match head powder, so this is possible to do.

## Describe the solution
Create Recipe to create a usable .22 casing with primer.
Change a name and description of .22 casings to match.

## Testing
grafik

### CODE REVIEW

File: data/json/items/generic/casing.json [hunk 1]
```diff
@@ -25,17 +25,17 @@
...
     "volume": "3 ml"
   },
   {
     "id": "22_casing_new",
     "copy-from": "22_casing",
     "type": "GENERIC",
-    "name": { "str": "unused .22 casing" },
-    "description": "An unfired, like-new .22 round casing, with the primer still intact.",
+    "name": { "str": "primed .22 casing" },
+    "description": "A like-new .22 round casing, with the primer inserted.",
```
Comment (chaosvolt) [inline data/json/items/generic/casing.json:38]: "inserted" might be taken to mean it's had a separate primer put in it, which isn't a thing with rimfire. Not sure what'd be a more clear wording though.

Comment (chaosvolt) [inline data/json/items/generic/casing.json:38]: Might work maybe?
```suggestion
    "description": "A like-new .22 rimfire casing, with fresh primer in the rim.",
```

File: data/json/items/generic/casing.json [hunk 2]
```diff
@@ -25,17 +25,17 @@
     "copy-from": "casing",
     "looks_like": "9mm_casing",
     "type": "GENERIC",
-    "name": { "str": ".22 LR casing" },
-    "description": "An empty casing from a .22 LR round.  These can't be hand-reloaded.",
+    "name": { "str": ".22 casing" },
+    "description": "An used, empty casing from a .22 round.",
```
Comment (chaosvolt) [inline data/json/items/generic/casing.json:29]: Also smol grammar fix.
```suggestion
    "description": "A used, empty casing from a .22 round.",
```

**Comment by RoyalFox2140** [comment]
This was brought to my attention and I really have nothing to say about this as I was going to simply finally do machining furniture and overhaul the heavy lathe and what not. I think you can safely merge this when others are satisfied because if I do something later with this it can just be reverted or overhauled again.

image

https://www.petersoncartridge.com/technical-information/drawing-brass/

**Comment by chaosvolt** [comment]
Whenever casing manufacture is added, this recipe can either be updated to be a full brass-working recipe plus a primer recipe, or we can just allow spent .22 casings to be made via a separate recipe and continue to leave it undefined just how hammered out the rim is, yeah.

**Review by chaosvolt — APPROVED** [review]

**Comment by Unrepentant-Atheist** [comment]
Wtf do you mean `reloading .22 with oxy` ??? People just go and reload a .22 with [Oxycotin](https://www.drugs.com/misspellings/oxycotin.html)???

**Comment by chaosvolt** [comment]
Oxidizer powder, as was already used in primer recipes. :V

**Pull request merged**